Sandbox/Main.cs

```diff
@@ -24,51 +23,42 @@ public partial class Main : Form
 
-            // Provided by Spice 3f5
-            double[] reft = new double[]
-            {
-                0.000000000000000e+00, 1.000000000000000e-11, 2.000000000000000e-11, 4.000000000000000e-11, 8.000000000000001e-11, 1.600000000000000e-10, 3.200000000000000e-10, 6.400000000000001e-10, 1.280000000000000e-09, 2.560000000000000e-09, 5.120000000000001e-09, 1.024000000000000e-08, 2.048000000000000e-08, 4.096000000000000e-08, 8.192000000000001e-08, 1.638400000000000e-07, 3.276800000000000e-07, 5.276800000000000e-07, 7.276800000000000e-07, 9.276800000000000e-07, 1.000000000000000e-06, 1.000100000000000e-06, 1.000300000000000e-06, 1.000505416909167e-06, 1.000916250727500e-06, 1.001000000000000e-06, 1.001040727433433e-06, 1.001122182300300e-06, 1.001285092034034e-06, 1.001610911501501e-06, 1.002262550436436e-06, 1.003565828306307e-06, 1.006172384046046e-06, 1.011385495525526e-06, 1.021811718484486e-06, 1.042664164402405e-06, 1.084369056238244e-06, 1.167778839909921e-06, 1.334598407253275e-06, 1.534598407253275e-06, 1.734598407253275e-06, 1.934598407253275e-06, 2.134598407253274e-06, 2.334598407253274e-06, 2.534598407253274e-06, 2.734598407253274e-06, 2.934598407253274e-06, 3.001000000000000e-06, 3.021000000000000e-06, 3.061000000000000e-06, 3.141000000000000e-06, 3.234987431524397e-06, 3.325439370636820e-06, 3.420856173579678e-06, 3.501000000000000e-06, 3.511070126950724e-06, 3.531210380852173e-06, 3.571490888655072e-06, 3.652051904260869e-06, 3.813173935472463e-06, 4.013173935472463e-06, 4.213173935472463e-06, 4.413173935472463e-06, 4.613173935472462e-06, 4.813173935472462e-06, 5.013173935472462e-06, 5.213173935472462e-06, 5.413173935472462e-06, 5.613173935472461e-06, 5.813173935472461e-06, 6.013173935472461e-06, 6.213173935472461e-06, 6.413173935472460e-06, 6.613173935472460e-06, 6.813173935472460e-06, 7.000000000000000e-06, 7.000100000000000e-06, 7.000300000000000e-06, 7.000505416909549e-06, 7.000916250728647e-06, 7.001000000000000e-06, 7.001040727433111e-06, 7.001122182299335e-06, 7.001285092031782e-06, 7.001610911496677e-06, 7.002262550426466e-06, 7.003565828286043e-06, 7.006172384005199e-06, 7.011385495443511e-06, 7.021811718320133e-06, 7.042664164073380e-06, 7.084369055579872e-06, 7.167778838592857e-06, 7.334598404618826e-06, 7.534598404618826e-06, 7.734598404618826e-06, 7.934598404618826e-06, 8.134598404618826e-06, 8.334598404618826e-06, 8.534598404618825e-06, 8.734598404618825e-06, 8.934598404618825e-06, 9.000999999999999e-06, 9.020999999999999e-06, 9.060999999999998e-06, 9.140999999999998e-06, 9.234987431524396e-06, 9.325439370636819e-06, 9.420856173579678e-06, 9.500999999999999e-06, 9.511070126950724e-06, 9.531210380852173e-06, 9.571490888655071e-06, 9.652051904260870e-06, 9.813173935472464e-06, 9.999999999999999e-06
-            };
-            double[] refv = new double[]
+            // Expected values:
+            // As simulated by ngSpice via PartSim
+            double[] reference = new double[]
```

Comment:
@svenboulanger Is it possible that you always create a new class for sandbox instead of overriding Main.cs ?

Reply:
I just edit the contents of the constructor in Main.cs. I don't think I'm doing anything else there...
For me it's okay to exclude the Sandbox project from the repository altogether. It's just used for quickly testing and debugging features anyway and is not really of any importance.

Reply:
Yes. You edit it. I see that Sandbox project is some kind of example project. It would be good to have all examples instead of the latest one.

Reply:
But in that case we are forced to maintain these examples. That is what the tests are for.